java-rest-service/config/workload-basic.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         autoscaling.knative.dev/minScale: "1"
     - name: testing_pipeline_matching_labels
       value:
-        apps.tanzu.vmware.com/pipeline: test-java
+        apps.tanzu.vmware.com/language: java
```

Comment:
Have we confirmed that the namespace provisioner provided Tekton pipelines have removed `apps.tanzu.vmware.com/pipeline: test-java`?  I believe they have, but just checking for verification.  Using only `apps.tanzu.vmware.com/language: java` will not work if the Tekton pipeline did not also removed `apps.tanzu.vmware.com/pipeline: test-java`

Reply:
this is a chicken or egg problem, maybe just leave the pipeline matching out